checks/container.go

```diff
@@ -159,9 +159,14 @@ func calculateCtrPct(cur, prev, sys2, sys1 uint64, numCPU int, before time.Time)
 		return 0
 	}
 
+	// Prevent uint underflows
+	if prev > cur || sys1 > sys2 {
```

Comment:
Should we be returning a negative percent here instead of zero?

Reply:
Spoke with Kevin offline, and I misunderstood what this was supposed to catch. 

Could this be more clearly written as: 

```
if cur == 0 || sys2 == 0 {
   return 0
}
```

Since zero is a value we're treating specially?

Reply:
For this case yes.  Given that the agent is more difficult to upgrade in the field, I'd prefer to be a bit defensive here though and guard against underflows in the case that the current values are non-zero and smaller than the previous ones which could occur in some future unforeseen circumstance.